activerecord/lib/active_record/database_configurations.rb

```diff
@@ -189,7 +189,7 @@ def method_missing(method, *args, &blk)
 
           ActiveRecord::Base.configurations = new_configs
         else
-          raise NotImplementedError, "`ActiveRecord::Base.configurations` in Rails 6 now returns an object instead of a hash. The #{method} method is not supported. Please use `configs_for` or consult the documentation for supported methods."
+          raise NotImplementedError, "`ActiveRecord::Base.configurations` in Rails 6 now returns an object instead of a hash. The `#{method}` method is not supported. Please use `configs_for` or consult the documentation for supported methods."
```

Comment:
Instead of saying `consult the documentation`, can we add the link to the documentation here?

Reply:
Also, can we rename `throw_setter_deprecation` -> `throw_setter_deprecation_warning` ??

Reply:
Docs get released when the version gets released so I can't yet link to the API docs.

Reply:
We're deleting this code in 6.1 - I think the current method name is fine.